### DISCUSSION TITLE
zinit drop me to a plugin directory after iTerm2 restored session from a system reboot

### DISCUSSION THREAD

**Discussion by 0x1b2c** [zdharma-continuum/zinit#139] [Q&A]
I've been using iTerm2's [session restoration feature](https://iterm2.com/documentation-restoration.html) for a long time, and have a habit of keep 9 tabs open so I can quickly switch between them for different jobs.

After a system reboot, though iTerm2 can't fully restore the session according to the above documentation, but it can always keep each tab in the original directory. But after I use zinit, reboot system may drop me to a plugin directory of zinit, as the screenshot shows.

<img width="920" alt="image" src="https://user-images.githubusercontent.com/13707/146214861-99d192e0-e4a2-4dd4-b503-30a728022083.png">

Not sure if this is a bug from zinit, so I'd like to ask this here.

**Comment by alichtman** [comment]
Seems like a zinit bug. Happens to me on Linux in gnome terminal too.

**Comment by 0x1b2c** [comment]
Thanks, I'll resubmit it as an issue.
